### PR TITLE
add channel concatenation function

### PR DESCRIPTION
**Pull request by mberz**
### Which issue(s) are closed by this pull request?

Closes #None

### Changes proposed in this pull request:

- Add a concatenation function, concatenating Signals along specified channels.

### CODE REVIEW

File: pyfar/signal.py [hunk 1]
```diff
@@ -726,3 +726,43 @@ def _divide(a, b):
...
+    data = np.concatenate(
+        (first._data, second._data),
+        axis=axis)
+
+    if first.comment is not None:
+        if second.comment is None:
+            comment = str(first.comment)
+        else:
+            comment = str(first.comment) + '\n' + str(second.comment)
+    else:
```
Comment (f-brinkmann) [inline pyfar/signal.py:760]: this case might not be catched:
`elif second.comment is not None: comment = str(second.comment)`

**Review by f-brinkmann — CHANGES_REQUESTED** [review]
see comment on `elif` case

**Comment by mberz** [comment]
May be meaningful to implement this as a class method rather than a function to avoid confusion with merges of other classes.

**Comment by twennemann** [comment]
We had the idea, to add an `ignore_axis` parameter in `utils.broadcast_scape`, so certain caxis will retain for concatenation.
Example:
We want to concatenate `signal1.cshape = (1, 2, 3)`  and `signal2.cshape = (1, 1, 2)`.                                                                                   
So `utils.broadcast_cshape(signal2, ignore_axis=-1)` will return `signal2.cshape = (1, 2, 2)`.
The concatenated signal will subsequently returned with `signal_conc.cshape = (1, 2, 5)`.

**Comment by mberz** [comment]
Closing, replaced by more recent PR #452.

**Pull request closed without merging**